histomicstk/features/ComputeGradientFeatures.py

```diff
@@ -96,32 +103,32 @@ def ComputeGradientFeatures(im_label, im_intensity,
         )
 
         # compute mean
-        fdata.at[i, 'MeanGradMag'] = np.mean(pixelGradients)
+        fdata.at[i, 'Gradient.Mag.Mean'] = np.mean(pixelGradients)
 
         # compute standard deviation
-        fdata.at[i, 'StdGradMag'] = np.std(pixelGradients)
+        fdata.at[i, 'Gradient.Mag.Std'] = np.std(pixelGradients)
 
         # compute skewness
-        fdata.at[i, 'SkewnessGradMag'] = scipy.stats.skew(pixelGradients)
+        fdata.at[i, 'Gradient.Mag.Skewness'] = scipy.stats.skew(pixelGradients)
 
         # compute kurtosis
-        fdata.at[i, 'KurtosisGradMag'] = \
+        fdata.at[i, 'Gradient.Mag.Kurtosis'] = \
             scipy.stats.kurtosis(pixelGradients)
 
         # compute intensity histogram
         hist, bins = np.histogram(pixelGradients, bins=num_hist_bins)
         prob = hist/np.sum(hist, dtype=np.float32)
 
         # compute entropy
-        fdata.at[i, 'HistEntropyGradMag'] = scipy.stats.entropy(prob)
+        fdata.at[i, 'Gradient.Mag.HistEntropy'] = scipy.stats.entropy(prob)
 
         # compute energy
-        fdata.at[i, 'HistEnergyGradMag'] = np.sum(prob**2)
+        fdata.at[i, 'Gradient.Mag.HistEnergy'] = np.sum(prob**2)
 
         bw_canny = cannyG[rprops[i].coords[:, 0], rprops[i].coords[:, 1]]
-        fdata.at[i, 'SumCanny'] = np.sum(bw_canny)
+        canny_sum = np.sum(bw_canny)
+        fdata.at[i, 'Gradient.Canny.Sum'] = canny_sum
 
-        fdata.at[i, 'MeanCanny'] = fdata.at[i, 'SumCanny'] / \
-            len(pixelGradients)
+        fdata.at[i, 'Gradient.Canny.Mean'] = canny_sum / len(pixelGradients)
```

Comment:
@cdeepakroy This Gradient.Canny.Mean goes to 0, need to be changed to float or ..